Sources/KSCrashRecordingCore/KSCPU.c

```diff
@@ -31,14 +31,34 @@
 
 #include <mach/mach.h>
 #include <mach-o/arch.h>
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 160000 // Xcode 14
```

Comment:
Maybe `#if __has_include(<mach-o/utils.h>)` here or this header may not have the necessary functions in older versions of the SDK?

Reply:
While effectively I indeed need to check for this file, I think it's better to just have this Xcode version check (where these functions were added)